cmd/edv-rest/startcmd/start.go

```diff
@@ -109,6 +115,15 @@ const (
 	logLevelInfo     = "info"
 	logLevelDebug    = "debug"
 
+	tlsSystemCertPoolFlagName  = "tls-systemcertpool"
```

Comment:
You also need to update the startup message log in the logStartupMessage function.